src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchConnectorIT.java

```diff
@@ -283,6 +283,21 @@ public void testBackwardsCompatibilityDataStream() throws Exception {
     container.start();
   }
 
+  @Test
+  public void testBackwardsCompatibilityDataStream2() throws Exception {
+    container.close();
+    container = ElasticsearchContainer.withESVersion("7.9.3");
```

Comment:
could we refactor this method and accept the esVersion as argument and then create two tests, one for 7.1 and another for 7.9.3?

Reply:
Not following -- we already have two tests -- one for 7.0.1 and one for 7.9.3 (it may show off the screen).  I can consolidate a ferw of the lines, I suppose, although they're pretty trivial to begin with.

Reply:
if we have test for 7.0.1 then no need to do what I am saying, I might not have realized this.